io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -114,7 +120,7 @@ private:
    std::vector<std::weak_ptr<TBufferMergerFile>> fAttachedFiles; //< Attached files
    std::function<void(void)> fCallback;                          //< Callback for when data is removed from queue
 
-   ClassDef(TBufferMerger, 0);
+   ClassDef(TBufferMerger, 1);
```

Comment:
Why change this? This only matters for classes that are streamed. This should have remained as 0.

Reply:
IIRC, when you have a ClassDef and you change data members you also should bump the ClassDef version.

Reply:
> IIRC, when you have a ClassDef and you change data members you also should bump the ClassDef version.

only when you change a persistent member :) ... and version == 0 means that all data members are implicitly marked transient.

Reply:
Shall I revert it back to 0?

Reply:
yes.

Reply:
Ok, done. I don't understand how is that a non persistent data member but you are the expert.